docs/strategy-advanced.md

```diff
@@ -49,7 +49,7 @@ from freqtrade.exchange import timeframe_to_prev_date
 
 class AwesomeStrategy(IStrategy):
     def confirm_trade_exit(self, pair: str, trade: 'Trade', order_type: str, amount: float,
-                           rate: float, time_in_force: str, sell_reason: str,
+                           rate: float, time_in_force: str, close_reason: str,
```

Comment:
This is a ... not so good change for now.

It's a user-facing interface - so by changing this we're breaking everyone's strategy that's using this.

I don't mind changing this _once it's necessary_ - but for now, i'd neglect this as it's a "theoretical" / behind the curtain change / PR, preparing things for short support - while not fully implementing it - so bothering users by breaking their stuff without any benefit / new feature will result in nothing but bad update / user experience and frustration.

This also applies to the database fields - renaming fields might break users custom analysis scripts - which is not necessary at this point. Neglecting this will also keep the diff quite small ... while now it's 16 files changed - where the "real" changes are in one file only.

Reply:
I changed it back